src/cogs/helpers/actions.py

```diff
@@ -268,7 +268,7 @@ async def challenge_selection(self):
 
         await self.ticket_channel.set_permissions(member, read_messages=True,
                                                   send_messages=None)
-        user_message = await self.ticket_channel.send("What have your tried so far?")
+        user_message = await self.ticket_channel.send("Please explain what you've tried so far and what you need help with")
```

Comment:
Looks a bit weird without punctuation imo

Reply:
you can add some

Reply:
Can't think of much rn.